src/mod/applications/mod_sms/mod_sms.c

```diff
@@ -610,6 +610,30 @@ SWITCH_STANDARD_CHAT_APP(reply_function)
 	return SWITCH_STATUS_SUCCESS;
 }
 
+
+SWITCH_STANDARD_CHAT_APP(transfer_function)
+{
+	char *context;
+
+	if (!data) return SWITCH_STATUS_SUCCESS;
```

Comment:
is it really supposed to return SWITCH_STATUS_SUCCESS if no data ? it seems wrong in multiple places in this file.

Reply:
not sure, i guess the correct way is print to user the missing data and return failure.  
and If failure it should stop executing the next actions, no?